tests/publiccloud/download_repos.pm

```diff
@@ -61,15 +61,16 @@ sub run {
         my $regex = "'s390x\\/|ppc64le\\/|kernel*debuginfo*.rpm|src\\/'";
         my $incident;
         for my $maintrepo (@repos) {
-            $incident = $1 if $maintrepo =~ /\/Maintenance:\/(\d+)/g;
+            $incident = $1 if $maintrepo =~ /\/(?>Maintenance|PTF):\/(\d+)/g;
             die "We did not detect incident number for URL \"$maintrepo\". We detected \"$incident\"" unless $incident =~ /\d+/;
+            # TODO: Please check manually the status of embargoed updates
             if (is_embargo_update($incident)) {
                 record_info("EMBARGOED", "The repository \"$maintrepo\" belongs to embargoed incident number \"$incident\"");
                 script_run("echo 'The repository \"$maintrepo\" belongs to embargoed incident number \"$incident\"'");
                 next;
             }
             script_run("echo 'Downloading $maintrepo ...' >> ~/repos/qem_download_status.txt");
-            my ($parent) = $maintrepo =~ 'https?://(.*)$';
+            my ($parent) = $maintrepo =~ 'https*://(.*)$';
```

Comment:
Why are we expecting more `s` here?

Reply:
Sorry.

Reply:
The `*` is still there, why do you want to replace the `?` (single optional occurrence) with a `*` (optional multiple occurrences)?

Reply:
Fixed now.

Reply:
Tested.